projects/lights_control/inc/lights_simple_peripherals.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+typedef enum {
+  LIGHTS_SIMPLE_STATE_ON = 0,
+  LIGHTS_SIMPLE_STATE_OFF,
+  NUM_LIGHTS_SIMPLE_STATES,
+} LightsSimpleState;
```

Comment:
What's this used for?

Reply:
it only was used in tests, moved it there.